gun_controller_topic:
  topic: /cmd_gun
  queue_size: 10

gpio_topic:
  topic: /cmd_gpio
  queue_size: 10
